tpl/images/images.go

```diff
@@ -117,3 +124,69 @@ func (ns *Namespace) Filter(args ...any) (images.ImageResource, error) {
 
 	return img.Filter(filtersv...)
 }
+
+type qrOptions struct {
+	Level     string
+	TargetDir string
+}
+
+var qrErrorCorrectionLevels = map[string]qr.Level{
+	"low":      qr.L,
+	"medium":   qr.M,
+	"quartile": qr.Q,
+	"high":     qr.H,
+}
+
+const qrDefaultErrorCorrectionLevel = "medium"
+
+// QR encodes the given text to a QR code using the given options, returning an
+// image resource.
+func (ns *Namespace) QR(args ...any) (images.ImageResource, error) {
+	if len(args) == 0 || len(args) > 2 {
+		return nil, errors.New("requires one or two arguments")
```

Comment:
We talked about making it simpler (to document etc.); I'm not sure allowing both positional args and a map accomplishes that.

Make it 1 arg (`opts any`).

Reply:
Make sense to me. Done.